ichnaea/api/locate/tests/test_fallback.py

```diff
@@ -522,38 +523,44 @@ def _query(self, **kwargs):
             kwargs["api_key"] = KeyFactory(fallback_cache_expire=60)
         return Query(**kwargs)
 
-    def test_get_blue(self, cache, stats):
+    def test_get_blue(self, cache, metricsmock):
         blues = BlueShardFactory.build_batch(2)
         query = self._query(blue=self.blue_model_query(blues))
         assert cache.get(query) is None
-        stats.check(
-            counter=[
-                ("locate.fallback.cache", 1, 1, [self.fallback_tag, "status:miss"])
```

Comment:
This has four values and they get unpacked as `(name, match, value, tags)` where "match" is the number of records that match those criteria. I'm not sure it matters here whether there's one record or more than one, so I didn't add that assertion.

Now I feel bad about that decision. I'll re-add it.